src/game/BattleGround/BattleGroundWS.cpp

```diff
@@ -553,10 +474,12 @@ WorldSafeLocsEntry const* BattleGroundWS::GetClosestGraveYard(Player* player)
     {
         if (GetStatus() == STATUS_IN_PROGRESS)
             return sWorldSafeLocsStore.LookupEntry(WS_GRAVEYARD_MAIN_ALLIANCE);
+
         return sWorldSafeLocsStore.LookupEntry(WS_GRAVEYARD_FLAGROOM_ALLIANCE);
     }
-    if (GetStatus() == STATUS_IN_PROGRESS)
+    else if (GetStatus() == STATUS_IN_PROGRESS)
```

Comment:
Redundant else, please don't undo my work: https://github.com/cmangos/mangos-tbc/commit/ea4256273ba5b82e85fd88d9ba155d75b98ea5fb#diff-9f4e0b9638611b3f7a54b42f2c8fa91e

Reply:
I second that.

Reply:
strange coding practices to remove an else  where even though logically it is still fine. The `else` from my experience also  makes it easier to read the code to understand the logical trail of the `if` statement. Removing them seems so pointless and deters away from the if statements logic trail. Doesn't make sense to remove them at all, unless I've missed something.

Reply:
1. Resharper and other code cleanup tools suggest it: https://www.jetbrains.com/help/resharper/RedundantIfElseBlock.html 2. It reduces nested code which you already noticed as you didn't add an else a few lines above/below either 3. It allows programmers to recognize this pattern quicker 4. It is currently consistent in the whole code base

Reply:
Seems to be semantics at this point, I'll change it back for the sake of point `4` which is a valid point.